mentoring/programs/shadow-roles.md

```diff
@@ -4,15 +4,16 @@ This area lists all of the shadow programs and how to get involved.
 Some areas are still in the process of forming shadow roles. 
 
 ## Release Team
-This SIG adheres to the Roles and Organization Management outlined in sig-governance and opts-in to updates and 
-modifications to sig-governance.
+This SIG release team is embedded within SIG Release and is responsible for the day to day work required to successfully release while the SIG at large is focused on the continued improvement of the release process.
+
+An overview of the release team can be found [here](https://github.com/kubernetes/sig-release/tree/master/release-team)
 
 The list of roles and duties can be found [here](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks)
 
 ## API Review
 
 ## Contributor Summit Planning
-The Content subteam of the events team is responsible for programming and content for periods of the contributor summit.
+The Content subteam creates in person and virtual experiences for new and current contributers to come learn, share and collaborate. 
```

Comment:
`contributers` is misspelled. It should be `contributors`. The test is failing for this. Please correct, push changes (don't forget to sign the commit.

Reply:
Does every commit need to be signed, or just the initial one?

Reply:
Every commit

Reply:
Ahh ok, will do so for next time. Sorry about that!